### DISCUSSION TITLE
Dapp run on webview in still connecting on click connect button

### DISCUSSION THREAD

**Discussion by KittinutzMuza** [WalletConnect/walletconnect-monorepo#3267] [Developer Support]
I implement the wallet that open localhost dapp by using web3modal run via webview on react native expo.
the modal open on the web browser is fine. 
But the modal do not open in react native webview.
It still connecting. 
Please provide me solution if anyone know how to fix it or explain it thank.
![Screenshot 2566-07-25 at 16 41 59](https://github.com/WalletConnect/walletconnect-monorepo/assets/134215736/0566f598-bba0-43eb-ba8d-7b4ac129aecc)

**Comment by glitch-txs** [comment]
Web3modal package is meant for websites only at the moment, a version for React Native is being developed, you can use this one in the meantime :)
https://github.com/WalletConnect/modal-react-native
